README.md

```diff
@@ -237,6 +237,38 @@ try {
 }
 ```
 
+### Toggle Climatisation
+
+To toggle climatisation mode to on or off. This call will return a `RemoteCommandAccepted` struct when the request has been accepted. 
+
+The `enable` paramater is optional and defaults to true. 
+
+```swift
+try {
+  let result = porscheConnect.toggleClimatisation(vehicle: vehicle, enable: false)
+  if let remoteCommandAccepted = result.remoteCommandAccepted {
+    // Do something with the remote command
+  }
+} catch {
+  // Handle the error
+}
+```
+
+As Toggle Climatisation is a remote command that can take time to reach and be executed by the car, you can check the status of the command. You pass in both the vehicle and the response from the `toggleClimatisation()` call above. 
+
+The `status` is mapped to a strongly typed enum that can be retrieved by accessing the `remoteStatus` calculated property.
```

Comment:
Consider documenting here that the status will eventually indicate success of the command.